library/testing/TestSDKHelpers.h

```diff
@@ -0,0 +1,22 @@
+#ifndef TestSDKHelpers_h
+#define TestSDKHelpers_h
+
+class TestSDKHelpers
+{
+public:
+  static bool RenderTest(f3d::window& win, const std::string& baseline, const std::string& output,
+    const std::string& diff)
```

Comment:
I think RenderTest(argv, win, name) could be a more specific and simpler approach